applications/crossbar/src/api_util.erl

```diff
@@ -1349,10 +1351,30 @@ do_create_resp_envelope(Context) ->
                    ]
            end,
 
-    kz_json:set_values(
-      props:filter_undefined(Resp)
-                      ,cb_context:resp_envelope(Context)
-     ).
+    encode_start_keys(kz_json:set_values(props:filter_undefined(Resp), cb_context:resp_envelope(Context))).
+
+-spec encode_start_keys(kz_json:object()) -> kz_json:object().
+encode_start_keys(Resp) ->
+    lists:foldl(fun(Key, JObj) ->
+                        case kz_json:get_value(Key, JObj) of
+                            'undefined' -> JObj;
+                            Value ->
+                                kz_json:set_value(Key, encode_start_key(Value), JObj)
+                        end
+                end
+               ,Resp
+               ,[<<"start_key">>, <<"next_start_key">>]
+               ).
+
+-spec encode_start_key(kz_json:path()) -> ne_binary().
+encode_start_key(StartKey) ->
+    kz_binary:hexencode(erlang:term_to_binary(StartKey)).
```

Comment:
why `kz_binary:hexencode` ? did you consider `kz_base64:encode` ? or `base64:encode` ?